src/operator/tensor/matrix_op-inl.h

```diff
@@ -601,18 +590,127 @@ void SliceCsrImpl(const SliceParam &param, const OpContext& ctx,
   });
 }
 
+// slice a CSRNDArray for two dimensions
```

Comment:
Let's add more documentation for the kernels like this one https://github.com/apache/incubator-mxnet/blob/master/src/operator/tensor/dot-inl.cuh#L40